contrib/babelfishpg_tsql/src/catalog.c

```
@@ -2090,6 +2140,24 @@ get_function_name(HeapTuple tuple, TupleDesc dsc)
 	return NameGetDatum(&(func->funcname));
 }
 
+static Datum
+get_perms_schema_name(HeapTuple tuple, TupleDesc dsc)
+{
+	bool		isNull;
+	Datum		schema_name = heap_getattr(tuple, Anum_bbf_schema_perms_schema_name, dsc, &isNull);
```

We should check the null value. IIUC, it shouldn't be null. We can throw an error if it is.